assets/src/edit-story/components/panels/design/border/borderWidth.js

```diff
@@ -65,9 +69,31 @@ const LabelText = styled.span`
 `;
 
 const ToggleWrapper = styled.div`
-  height: ${({ locked }) => !locked && '60px'};
+  padding-top: 2px;
+  ${({ locked }) =>
+    locked &&
+    css`
+      padding-left: 8px;
+    `};
+  align-self: stretch;
```

Comment:
@miina, cleaned this up a bit too. The old syntax relied on `height: false` being removed from the output (which it is, but still a bit hacky).

Reply:
Thank you!